docs/TOOLS.md

```diff
@@ -0,0 +1,57 @@
+## Code Style and Linting
+
+If you are looking for good general advices regarding Perl coding, 
+Damian Conway's [Perl Best Practices][PBP] is a good reference. 
```

Comment:
I'll try to find something. It's... been a while since I had to read about Perl coding recommendations. :-)

Of course, one can read the source of the PBP perlcritic rules, which is pretty much a condensed version of the book.